package.json

```diff
@@ -7,6 +7,7 @@
     "build": "eleventy",
     "lint": "eslint .",
     "serve": "rm -rf _site/ && eleventy --serve",
+    "start": "rm -rf _site/ && eleventy --serve",
```

Comment:
Are both of these needed?

Reply:
I historically use `npm run serve` which is popular in the static site community. We use `npm start` here a lot. This accommodates both. Is there harm in having them both?

Reply:
A DRY issue.  Could call one from the other too.  It's one line, it's fine.
A DRY issue.  Could call one from the other too.  It's one line, it's fine.